### ISSUE TITLE
Allow import declarations anywhere

### ISSUE THREAD

**Issue by sameera-jayasoma** [ballerinalang/ballerina#2399]
function foo () {
  import a.b.c as x;

  // use stuff from x here
  x:doSomething (...);
}

**Comment by afkham** [comment]
@sanjiva  @sameera-jayasoma  Do we really need this? When all imports are grouped in a single location, the code is much cleaner.

**Comment by sanjiva** [comment]
Yeah its ok to close this .. but I kinda feel localizing imports is good namespace management discipline - in Java code you often see a ton of imports and some imports may only be used in one place; that's not good coding discipline.

The only challenge is the compiler needs to maintain dependencies but its not a massive burden.
